sudoku/sudoku.py

```diff
@@ -0,0 +1,147 @@
+# solve sudoku puzzle
+
+
+def solve_board(input_board):
+    """Solve a sudoku puzzle
+
+    Args:
+        input_board (list of list[int]): list of lists with integers
+        representing the sudoku board
+
+    Returns:
+        list of list: solved sudoku board else prints unsolvable board
+    """
+    board = input_board.copy()
+    if solve_helper(board):
+        return board
+    else:
+        print(" Board cannot be solved  ....!")
```

Comment:
Don't print things. Its not helpful or useful as an API. The calling function is not going to read Stdout.